Doc/whatsnew/3.14.rst

```diff
@@ -109,6 +109,8 @@ pathlib
 * Add :meth:`pathlib.Path.copytree`, which copies one directory tree to
   another.
   (Contributed by Barney Gale in :gh:`73991`.)
+* Add :meth:`pathlib.Path.rmtree`, which recursively removes a directory.
```

Comment:
This comment is valid for the above two other additions (yours) but you can use 

```rst
:meth:`Path.rmtree <pathlib.Path.rmtree>`
```

This will render as `Path.rmtree` but will correctly resolve to `pathlib.Path.rmtree`. I think it would be clearer for the reader since we already know that we are using objects in the `pathlib` module.

Reply:
I've done this a slightly different way - let me know what you think?